models/constants.go

```diff
@@ -20,3 +20,8 @@ const ServiceName = "cas-scheduler"
 const (
 	Env_EnvTag = "ENV_TAG"
 )
+
+// Metrics
+// Counts
+const CreatedBatchMetricName = "created_batch"
```

Comment:
Should we make these into an enum?
```
type MetricName string

const (
	MetricName_CreatedBatch    MetricName = "created_batch"
	MetricName_ReplacedRequest MetricName = "replaced_request"
)

```

Reply:
Do we need additional metrics?
- Ingress requests for the Validation service so we can see how much load we're receiving
- Ingress requests to the Batching service so we can see how many requests got past deduplication
- Avg. number of requests in a Batch so we can see the pattern of batch sizes over time under different loads
- Number of anchor jobs created by the Worker service
- Number of failures received by the Failure Handler
- Others?

Don't have to implement all these in this PR though we should track this list somewhere and add them in subsequent PRs.

Reply:
Also question for @gvelez17 here - when we replace the CASv2 Scheduler with the CASv5 Scheduler, will we lose any metrics you might have added to the old Scheduler? If so, then we probably need to make sure that at least those metrics, if still useful in the new design, are retained in the new code.

Reply:
I'll make a ticket